index.rst

```diff
@@ -48,14 +48,480 @@
+Thus, we can still detect and measure sources in the difference image, though we have far more to deal with than for a typical observation.
+Most of the residuals are also unfortunately typical of those we see in less crowded fields, which is indicative of our need to improve the implementation our image differencing algorithm in general.
+The improvements that the Alert Production team is currently working on should result in better subtractions for crowded fields as well.
+
+
+..
+  Density of DIA sources on a single ccd
+  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+  .. figure:: /_static/DiaSource_density_B1_g_ccd42.png
+   :name: dia_source_density_B1_g
+  
+   Density of detected DIA sources across all visits for field B1 in g-band, for ccd 42.
+   Compare to :numref:`source_density_B1_g` for the number of sources in the original images.
+  
+  .. figure:: /_static/DiaSource_density_B2_g_ccd42.png
+   :name: dia_source_density_B2_g
+  
+   Density of detected DIA sources across all visits for field B2 in g-band, for ccd 42.
+   Compare to :numref:`source_density_B2_g` for the number of sources in the original images.
+  
+  .. figure:: /_static/DiaSource_density_B1_i_ccd42.png
+   :name: dia_source_density_B1_i
+  
+   Density of detected DIA sources across all visits for field B1 in i-band, for ccd 42.
+   Compare to :numref:`source_density_B1_i` for the number of sources in the original images.
+  
+  .. figure:: /_static/DiaSource_density_B2_i_ccd42.png
+   :name: dia_source_density_B2_i
+  
+   Density of detected DIA sources across all visits for field B2 in i-band, for ccd 42.
+   Compare to :numref:`source_density_B2_i` for the number of sources in the original images.
+
+Conclusions and future work
+===========================
+
+This investigation has stress-tested the LSST Science Pipelines, and uncovered several algorithmic components that need attention.
+Some of those improvements, such as upgrading the PSF determiner, were necessary to process the data and have already been completed.
+Others, such as the fidelity of image differencing, had been previously identified and the improvements are under active development.
+
+Summary of the challenges to processing crowded fields identified in this analysis:
+
+- The PSF determiner was upgraded to PSFex, which is able to run on crowded fields. However, it does not appear to be able to model the wings of the PSF (see :numref:`psf_B1_2013_g` through :numref:`psf_B2_2015_i`).
+- The cosmic ray detection and repair algorithm still fails for some ccds, and will require either careful tuning of the existing parameters or a more sophisticated implementation.
+- Photometric calibration is at times inconsistent, with offsets of several magnitudes in the worst cases (:numref:`source_counts_2013_B1_g` - :numref:`source_counts_2015_B2_i`). This is likely to improve with a better PSF model.
```

Comment:
It's not clear to me that PSF modeling would cause the problems with photometric calibration--let's not speculate here.